refarch-integrations/refarch-s3-integration/refarch-s3-integration-client-core/src/main/java/de/muenchen/refarch/integration/s3/client/repository/transfer/S3FileTransferRepository.java

```diff
@@ -221,6 +187,14 @@ public void deleteFile(final String presignedUrl)
                     HttpMethod.DELETE,
                     fileHttpEntity,
                     Void.class);
+            return null;
+        });
+    }
+
+    private <T> T executeWithErrorHandling(final Callable<T> callable)
```

Comment:
Is this method only relevant for this class or can be maybe also reuse it in other places?

Reply:
The exceptions are not always the same and also this would link different integrations together.